### PR TITLE
Added support to use testenv

### PR DESCRIPTION
**Pull request by vijetmahabaleshwar-okta**

### CODE REVIEW

File: e2e-tests/tools/common-config/index.js [hunk 1]
```diff
@@ -14,6 +14,9 @@
  * limitations under the License.
  */
 /* global jasmine */
+const path = require('path');
+// Env vars on new lines in the form of NAME=VALUE in testenv file at root of project
+require('dotenv').config({path: path.join(__dirname, '..', '..', '..', '..', 'testenv')});
```
Comment (vijetmahabaleshwar-okta) [inline e2e-tests/tools/common-config/index.js]: Any sample/library that wants to run e2e tests can now provide env vars at the root of the project in a file testenv

**Comment by bretterer** [comment]
@bdemers I agree with everything you say here, but for testing, I am personally ok with providing it in the project with .gitignore updated.. Maybe we can provide both options?

**Comment by robertjd** [comment]
@vijetmahabaleshwar-okta is the idea here that using a file will be simpler than environment variables for our cross-platform needs?

**Comment by bdemers** [comment]
My view is that secrets in files should be minimized (and a single file in `~/.some-where` is easier to protect. But as long as the file gets added to a `.gitignore` i will not object

**Comment by vijetmahabaleshwar-okta** [comment]
@robertdamphousse-okta - Yes, that is the idea. @bretterer has been using a `testenv` file in his samples, instead of running the `export` commands.

@bdemers - Using `testenv` won't be enforced on any repo. I can also look for `~/.okta/testenv` first if it exists. Will definitely update the `.gitignore`

**Pull request merged**